airflow/settings.py

```diff
@@ -432,7 +432,12 @@ def import_local_settings():
 
         log.info("Loaded airflow_local_settings from %s .", airflow_local_settings.__file__)
     except ImportError:
-        log.debug("Failed to import airflow_local_settings.", exc_info=True)
+        log.warning("Failed to import airflow_local_settings.", exc_info=True)
+    except ModuleNotFoundError as e:
```

Comment:
ModuleNotFoundError derirves from ImportError so it should be higher.

Reply:
Done. Also made it critical and re-raise the exception.

This is no different that other possible errors, like `SyntaxError`.